lib/CMakeLists.txt

```diff
@@ -304,13 +331,14 @@ if(NOT VIPS)
       --with-tiff
       --with-tiff-includes=${EXT_INSTALL_DIR}/include
       --with-tiff-libraries=${EXT_INSTALL_DIR}/lib
+      --with-cgif
       --without-magick
       --without-orc
       --without-gsf
       --without-rsvg
       ${LIBSPNG_FLAGS}
       ${LIBHEIF_FLAGS}
-      DEPENDS libjpeg libpng libspng giflib libwebp libimagequant lcms2 libheif tiff
+      DEPENDS libjpeg libpng libspng giflib cgif libwebp libimagequant lcms2 libheif tiff
```

Comment:
Ok, I see that `giflib` is a decoder and `cgif` an encoder. I'd like to make sure that we can decode/encode/decode .gif files without any issue.

Reply:
@warrenseine I've enabled GIFs in all of the tests, so I think they are confirming that?